### ISSUE TITLE
Use of uninitialized value $err in concatenation

### ISSUE THREAD

**Issue by thomasmitchell** [genesis-community/genesis#416]
```
Task 174872 Started  Wed May 13 13:51:10 UTC 2020
Task 174872 Finished Wed May 13 13:53:38 UTC 2020
Task 174872 Duration 00:02:28
Task 174872 done

Succeeded
[WARNING] The local-db flag has been renamed to local-postgres-db
[WARNING] The local-blobstore feature is now the default behaviour and doesn't need
          to be specified in the environment file

Update your snw-tmitchell-lab.yml file to remove these warnings


Deployment successful.  Preparing metadata for export...
Connecting to Credhub to extract secrets...Use of uninitialized value $err in concatenation (.) or string at /home/klaue/.geese/lib/Genesis/Env.pm line 866.
failed!

[ERROR] Could not target credhub specified by snw-tmitchell-lab/bosh
```

uninitialized variable error output.

**Comment by thomasmitchell** [comment]
expected error output is `'bash: credhub: command not found`

**Comment by dennisjbell** [comment]
This is actually two issues: 
1) cli command should be detected by genesis (similar to bosh, safe, jq, ...)
2) error message was missing and not conveyed to user

**Comment by mrferris** [comment]
Fixed in commit 0c7ef5466551c56dc25e6edce20cf1c540b2e932
